### PR TITLE
Fixing graph client submodules

### PR DESCRIPTION
**Pull request by liambeckman199**

### CODE REVIEW

**Comment by ardatan** [comment]
Thanks for the PR but could you add some description and give some context?

File: examples/cross-chain-extension/.graphclientrc.yml [hunk 1]
```diff
@@ -1,24 +0,0 @@
-sources:
-  - name: Sushiswap
```
Comment (ardatan) [inline examples/cross-chain-extension/.graphclientrc.yml:2]: Why do we need to delete this?

File: examples/messari-graphclient/.graphclientrc.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+additionalTypeDefs: |
+  extend type LiquidityPool {
+    protocolName: String
+  }
+  extend type Query {
+    crossLiquidityPools(first: Int!, protocolNames: [String!]!, orderBy: BigDecimal!, orderDirection: OrderDirection!, where: LiquidityPool_filter!): [LiquidityPool!]!
+  }
+
+  extend type LiquidityPoolDailySnapshot {
```
Comment (ardatan) [inline examples/messari-graphclient/.graphclientrc.yml:16]: You can have one `extend type` for all field extensions.

File: examples/messari-graphclient/src/resolvers-backup.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                //const RewardsROI_result =
+                //  snapshot.RewardsPerToken !== undefined && snapshot.RewardsPerToken.length > 0
+                //    ? snapshot.RewardsPerToken.map((i) => {
+                //        return i / snapshot.outputTokenPriceUSD
+                //      })
+                //    : undefined
+
+                return {
+                  ...snapshot,
+                  FeesUSD: FeesUSD_result,
```
Comment (ardatan) [inline examples/messari-graphclient/src/resolvers-backup.ts:71]: You can have additionalResolvers for the extra fields here instead

File: examples/messari-graphclient/src/resolvers-backup.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+                  TokenFeesRaw: TokenFeesRaw_result,
+                  RewardsPerToken: RewardsPerToken_result,
+                  //RewardsROI: RewardsROI_result,
+                }
+              })
+
+              return {
+                ...liquidityPool,
+                protocolName,
+                dailySnapshots,
```
Comment (ardatan) [inline examples/messari-graphclient/src/resolvers-backup.ts:82]: If you use additional resolvers for the type of `snapshot`, you don't need any of this.

File: examples/cross-chain-extension/.gitignore [hunk 5]
```diff
@@ -1 +0,0 @@
-.graphclient
```
Comment (ardatan) [inline examples/cross-chain-extension/.gitignore:1]: Why do we need to delete this?

File: examples/messari-graphclient/daily-query.graphql [hunk 6]
```diff
@@ -0,0 +1,55 @@
+query CrossRebasesExample($protocolNames: [String!]!, $tokenIDs: [String!]!) {
```
Comment (ardatan) [inline examples/messari-graphclient/daily-query.graphql:1]: Operation names should be unique

File: examples/messari-graphclient/src/resolvers-backup.ts (hunk 4 above)
Comment (ardatan) [reply examples/messari-graphclient/src/resolvers-backup.ts:82]: Please check;
https://github.com/graphprotocol/graph-client/pull/142

**Comment by liambeckman199** [comment]
> Thanks for the PR but could you add some description and give some context?

My apologies, accidentally pushed to main. I appreciate the comments and feedback though!

**Comment by ardatan** [comment]
I think we can close this :) @liambeckman199 Please feel free to create issues if you have any problems.

**Pull request closed without merging**